packages/grafana-ui/src/themes/GlobalStyles/page.ts

```diff
@@ -117,5 +117,79 @@ export function getPageStyles(theme: GrafanaTheme2) {
       margin-left: ${theme.spacing(1)};
       margin-top: ${theme.spacing(0.5)};
     }
+
+    .chat-wrapper {
```

Comment:
:thinking: in general -- we try to avoid adding raw css to global styles.  Ideally this can be moved to the component that will use the styles